misk-aws2-dynamodb-testing/src/test/kotlin/misk/aws2/dynamodb/testing/AbstractDynamoDbTest.kt

```diff
@@ -0,0 +1,143 @@
+package misk.aws2.dynamodb.testing
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbEnhancedClient
```

Comment:
does enhanced mean v2?

Reply:
There's also a non-enhanced client that's also in v2. Anything in the software.amazon.awssdk.* package is v2.